pkg/network/sriov/sriov_test.go

```diff
@@ -29,7 +31,12 @@ import (
 )
 
 var _ = Describe("SRIOV", func() {
-	networkStatusWithOneSRIOVNetwork := `
+
+	const (
+		booVMNetworkPodIfaceName = "net6446d58d"
+		fooVMNetworkPodIfaceName = "net2c26b46b"
```

Comment:
nit: IMO we should keep the humorist naming to lectures (foo & boo), but it is up to you.

Reply:
I don't follow; I'm naming the interfaces after the names of the networks to which they attach. 

I'm open to alternatives, but, imo, I don't have a better reference for each interface @EdDev